src/pemfile.rs

```diff
@@ -22,6 +22,11 @@ pub enum Item {
     /// Appears as "CERTIFICATE" in PEM files.
     X509Certificate(CertificateDer<'static>),
 
+    /// A DER-encoded Subject Public Key Info; as specified in RFC 7468.
+    ///
+    /// Appears as "PUBLIC KEY" in PEM files.
```

Comment:
Is there an RFC or IANA registry or something to point to for this fact?

Reply:
Oh derp, it's right above: [RFC 7468, Section 13](https://datatracker.ietf.org/doc/html/rfc7468#section-13) - "Textual Encoding of Subject Public Key Info"